webapp/portlet/src/main/webapp/vue-apps/space-top-bannerlogo/components/ExoSpaceLogoBanner.vue

```diff
@@ -16,14 +16,14 @@
           v-on="on"
           v-bind="attrs"
           class="d-inline-flex">
-          <a :href="portalPath">
+          <a :href="portalPath" :aria-label="$t('space.logo.banner.title',{0:logoTitle})">
             <v-list-item-avatar 
               v-if="logoPath"
               id="UserHomePortalLink"
               size="30"
               class="tile my-0 spaceAvatar ms-0 me-3"
               tile>
-              <v-img :src="logoPath" :alt="logoTitle" />
+              <img :src="logoPath" :alt="$t('space.logo.banner.alt',{0:logoTitle})">
```

Comment:
Why did you change to img instead of v-img ?

Reply:
When converting the view tag to html tag the alt props will be converted to arial-label and in the task it asks me to add an alt props in image tag